trecs/models/recommender.py

```diff
@@ -260,6 +267,7 @@ def update_predicted_scores(self):
         user_profiles = self.users_hat
         item_attributes = self.items_hat
         predicted_scores = self.score_fn(user_profiles, item_attributes)
+        # import pdb; pdb.set_trace()
```

Comment:
just curious, what was this for? Should it stay?

Reply:
Oops, nope! Got rid of it!

Reply:
it was for aiding w/ debugging :)